blockchain/fullblocktests/generate.go

```diff
@@ -2546,6 +2736,77 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.AssertTipNumRevocations(1)
 	accepted()
 
+	// Create block that has a revocation with more payees than expected.
+	//
+	//   ... -> brt1(24) -> brt4(25)
+	g.SetTip("brt1")
+	g.NextBlock("brt4", outs[25], ticketOuts[25], func(b *wire.MsgBlock) {
```

Comment:
The current `brt3` needs to be moved below all of these (and probably renamed to something like `brtfinal`.  Otherwise, you're trying to put these blocks on a side chain since the current best chain is already at the tip of `brt3`.

e.g. You're doing:

```Go
//   ... -> brt1(24) -> brt3(25)
//                  \-> brt4(25)
```

Which is not what you intend.